airbyte-integrations/connectors/source-github/source_github/graphql.py

```diff
@@ -87,12 +93,113 @@ def get_query_reviews(owner, name, first, after, number=None):
     )
     reviews.nodes.commit.oid()
     user = reviews.nodes.author(__alias__="user").__as__(_schema_root.User)
-    user.__fields__(
-        id="node_id",
-        database_id="id",
-        login=True,
-        avatar_url="avatar_url",
-        url="html_url",
-        is_site_admin="site_admin",
-    )
+    select_user_fields(user)
     return str(op)
+
+
+class QueryReactions:
+
+    AVERAGE_REVIEWS = 5
+    AVERAGE_COMMENTS = 2
+    AVERAGE_REACTIONS = 2
+
+    def get_query_root_repository(self, owner, name, first, after=None):
+        op = sgqlc.operation.Operation(_schema_root.query_type)
```

Comment:
I see we use this line in a lot of methods. May be better shift it to class or instance attribute.